src/AIS_Decoder.cpp

```diff
@@ -2238,14 +2237,11 @@ void AIS_Decoder::OnTimerAIS( wxTimerEvent& event )
                     current_targets->erase( it );
```

Comment:
hi,
It's configuration dependant, but I think that after this line 'it'  is undefined and you can't do 'i++' later.

cf:
http://stackoverflow.com/questions/4600567/how-can-i-delete-elements-of-a-stdmap-with-an-iterator